js/src/chrome/projects/chrome.projects.js

```diff
@@ -645,13 +676,23 @@ Wu.Chrome.Projects = Wu.Chrome.extend({
 				if (options.type == 'read' && user.getUuid() == app.Account.getUuid()) return;
 
 				// add selected user item to input box
+				me._checkedUsers[options.type][user.getFullName()] = user;
 				this._addUserAccessItem({
 					input : invite_input,
 					user : user,
-					type : options.type
+					type : options.type,
+					checkedUsers : me._checkedUsers[options.type],
+					onKeyUp: onKeyUp
```

Comment:
then you pass the function here, `onKeyUp : this._onKeyUp` (or `_onKeySomething` or whatever)